sites/fast-website/src/app/components/content-placement-container/content-placement-container.template.ts

```diff
@@ -7,10 +7,15 @@ export const ContentPlacementContainerTemplate = html<ContentPlacementContainer>
         html<ContentPlacementContainer>`
             ${repeat(
                 x => x.frameworkContentPlacementData,
-                html`<site-content-placement class="framework_ContentPlacement">
-                    <h3>${x => x.header}</h3>
-                    <p slot="body">${x => x.body}</p>
-                </site-content-placement>`
+                html`
+                    <site-content-placement class="framework_ContentPlacement">
+                        <h3>
+                            ${x => (x.headerSubscript ? x.header + " " : x.header)}
+                            <span class="headerSubscript">${x => x.headerSubscript}</span>
```

Comment:
Can we make this a `<small>`?

Reply:
Yes. I have changed it over to small, but I still seem to need to use the type-ramp to get the size that we desire.